arviz/tests/test_data.py

```diff
@@ -30,6 +34,60 @@ def chains():
     return 2
 
 
+@pytest.fixture(autouse=True)
+def no_remote_data(tmpdir):
+    """Delete all remote data and replace it with a local dataset."""
+    keys = list(REMOTE_DATASETS)
```

Comment:
Not required. This would be a good candidate for use of the `monkeypatch` fixture. This code effectively does the same thing so it's not a concern. Just wanted to mention it.